code/_globalvars/lists/mobs.dm

```diff
@@ -32,7 +32,7 @@ GLOBAL_LIST_EMPTY(ai_list)
 GLOBAL_LIST_INIT(simple_animals, list(list(),list(),list(),list())) // One for each AI_* status define
 GLOBAL_LIST_EMPTY(living_cameras)
 GLOBAL_LIST_EMPTY(aiEyes)
-GLOBAL_LIST_EMPTY(humans_by_zlevel)			//z level /list/list of alive humans
+GLOBAL_LIST_EMPTY_TYPED(humans_by_zlevel, /list/mob/living/carbon/human)			//z level /list/list of alive humans
```

Comment:
why's this glob need to be typed compared to the others?

Reply:
its a list of lists and sdmm complains because of the ambiguous type inferring here https://github.com/tgstation/TerraGov-Marine-Corps/blob/754ea07b335ebad9582ca7bf6f963ac99551b02c/code/controllers/subsystem/server_maint.dm#L50-L51

Reply:
ah cool